### ISSUE TITLE
The grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16 test fails in run_MET_Pb2nc_obs on Cheyenne GNU

### ISSUE THREAD

**Issue by MichaelLueken** [ufs-community/ufs-srweather-app#817]
## Expected behavior
Fundamental tests should successfully pass on all Tier-1 platforms.

## Current behavior
The `grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16` test is failing in `run_MET_Pb2nc_obs` due to a missing shared library/object file:

`/glade/p/ral/jntp/MET/MET_releases/10.1.1/bin/pb2nc: error while loading shared libraries: libimf.so: cannot open shared object file: No such file or directory`

It should be noted that this is only an issue for Cheyenne GNU.  There are no issues with the test on Cheyenne Intel.

## Machines affected
Cheyenne using the GNU compiler

## Steps To Reproduce
1. Clone the current develop on Cheyenne
2. Compile using GNU compiler - `./devbuild.sh -p=cheyenne -c=gnu`
3. Run the `grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16` test - 
`./run_WE2E_tests.py -t=grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16 -m=cheyenne -c=gnu -a=********`
4. See the failure in the `metplus.log.Pb2nc_obs_2021051212` log file.

**Comment by MichaelLueken** [comment]
@christopherwharrop-noaa From the comment that you left in the SRW CM meeting chat today, it appears as though METplus was built on both Hera and Cheyenne using the Intel compiler only.  For Hera GNU, this was addressed by forcing an Intel compiler to be loaded before running the verification tasks.  I was hoping that adding a `modulefiles/tasks/cheyenne/run_vx.local.lua` and similarly loading the Intel compiler used to build METplus would get the `grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16` test to work, but there is something going on with the module purge that is supposed to happen before the local task modulefiles are loaded.  After the module purge, the module is switched from `gnu/11.2.0` to `intel/2021.2`, rather than directly loading `intel/2021.2`.  Even the current Cheyenne Intel test is loading the `build_cheyenne_intel.lua` modulefile before loading the necessary conda environment.  I'm not sure what is going on at the moment, but further investigation is required.

**Comment by natalie-perlin** [comment]
working on it now.

**Comment by MichaelLueken** [comment]
@natalie-perlin I have opened PR #820 to address this issue.  On Hera, the method to make this work was to first load the `build_hera_gnu` modulefile, then load an Intel compiler, and finally load the `python_srw` modulefile.  On Cheyenne, it was noted that after loading the `build_cheyenne_gnu` modulefile, attempting to load an Intel compiler would cause the modules to attempt to replace the `gnu/11.2.0` with the chosen Intel compiler, causing the script to fail.  So, I had to create a run_vx.local.lua modulefile that first unloads `build_cheyenne_gnu`, then loads the `intel/2021.2` compiler, and then load `python_srw`.  This allows the test to run on Cheyenne GNU.  I'm certainly open to opinions or, if you would prefer, I can close my PR if you wish to open your own.  Either option works for me.
